dist/api_err.py

```diff
@@ -208,6 +208,17 @@ def __init__(self, name, value, desc, long_desc=None, **flags):
 
 \treturn (__wt_strerror(NULL, error, buf, sizeof(buf)));
 }
+
+/*
+ * __wt_is_valid_sub_level_error --
+ *\tReturn true if the provided error falls within the valid range for sub level error codes, 
```

Comment:
It looks like you've got some tab characters in the source for indenting - these should be spaces.

Reply:
Ah ok, I was following the style of the surrounding lines but I can change it. Should I replace all the tabs in `api_err.py` with spaces?

Reply:
Actually, I just noticed that the rest of that file is using '\t' to indicate tabs, so please ignore that comment of mine.